lib/pbench/server/database/models/dataset.py

```diff
@@ -341,7 +341,7 @@ class Dataset(Database.Base):
         transition  The timestamp of the last state transition
     """
 
-    __tablename__ = "datasets"
+    __tablename__ = "dataset"
```

Comment:
I'm not thrilled about changing all the table names, but I'll pre-resolve this just because it's not a big deal and I know there are differing style guidelines and ways to look at this. Basically, I started using plural table names because it looks better semantically in SQL statements (which we rarely use directly), as in you're selecting from a set of rows, plural: "select * from datasets".

Reply:
I'd be happy to leave the table name plural and I'll add a comment.

Reply:
Posted an update with the changes.